libraries/devsec_ssh.rb

```diff
@@ -177,6 +184,13 @@ def find_ssh_version(version, versions)
         found_ssh_version
       end
 
+      { client: 'sshclient',
```

Comment:
I'm worried I may have damaged this section: in runningman84's branch, it sits in the `private` section of the lib, where I'm putting it now, but master appears to have it in the "main" class. I put it back in private but I might need someone to take a look pls (I'll fix it of course if needed)